io.github.xplshn.Openarena.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.xplshn.Openarena
```

Comment:
This should probably be ws.openarena.OpenArena.

Reply:
No because the page of the original OpenARENA devs is mostly dead, they haven't made a release since 2012, that's why I'm making a release based on the up to date Quake3e engine.

Reply:
Look, http://www.openarena.ws/smfnews.php I can't use this ID because I'm not affiliated to the devs.